.github/workflows/pr-check.yaml

```
@@ -542,6 +542,11 @@ jobs:
             ./tests/**/output/
             !./tests/**/traces/raw
 
+  managed-configuration:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'area/ci') || !github.event.pull_request.draft }}
+    uses: ./.github/workflows/managed-configuration-template.yaml
+    continue-on-error: true
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "managed-configuration"

(syntax-check)